source/_docs/scripts/conditions.markdown

```diff
@@ -239,10 +240,31 @@ Note that if only `before` key is used, the condition will be `true` *from midni
 If only `after` key is used, the condition will be `true` from the specified time *until midnight*.
 Time condition windows can span across the midnight threshold if **both** `after` and `before` keys are used. In the example above, the condition window is from 3pm to 2am.
 
-<div class='note tip'>
+<div class='note warning'>
+As of release 0.109, the weekday property will be deprecated. Replace it with weekday condition.
+</div>
 
+<div class='note tip'>
 A better weekday condition could be by using the [Workday Binary Sensor](/integrations/workday/).
+</div>
 
+### Weekday condition
+
+The weekday condition can test if it is a certain day of the week.
+
+```yaml
+condition:
+  condition: weekday
+  days:
+    - mon
+    - wed
+    - fri
+```
+
+Valid values for `days` are `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
+
+<div class='note tip'>
```

Comment:
Same comment as above... do we need to mention this twice btw?

Reply:
fixed